v2/sample-app/app/libs/styles.xml

```diff
@@ -262,5 +262,58 @@
         <item name="android:background">@drawable/ic_baseline_check_circle_24</item>
     </style>
 
+    // consent feature related styles
+    <style name="socure_consent_title">
+        <item name="fontFamily">@font/proxima_nova_bold</item>
+        <item name="android:textColor">@color/socure_title_text</item>
+        <item name="android:textStyle">bold</item>
+        <item name="android:textSize">24sp</item>
+        <item name="android:maxLines">3</item>
+        <item name="android:ellipsize">end</item>
+        <item name="android:gravity">center_horizontal</item>
+    </style>
+
+    <style name="socure_small_regular_text">
+        <item name="fontFamily">@font/proxima_nova_regular</item>
+        <item name="android:textColor">@color/socure_dark_blue</item>
+        <item name="android:textSize">16sp</item>
+        <item name="android:gravity">left</item>
+        <item name="android:linksClickable">true</item>
+    </style>
+
+    <style name="socure_btn_orange_bg" parent="Widget.AppCompat.Button.Borderless">
+        <item name="fontFamily">@font/proxima_nova_bold</item>
+        <item name="android:textStyle">bold</item>
+        <item name="theme">@style/Widget.AppCompat.Button.Borderless</item>
+        <item name="android:textSize">16sp</item>
+        <item name="android:layout_gravity">center</item>
+        <item name="android:gravity">center</item>
+        <item name="textAllCaps">false</item>
+        <item name="contentPadding">2dp</item>
+        <item name="android:maxLines">1</item>
+        <item name="android:ellipsize">end</item>
+        <item name="android:background">@drawable/btn_socure_orange_bg</item>
+        <item name="android:textColor">@color/socure_white</item>
+    </style>
+
+    <style name="socure_btn_orange_stoke" parent="Widget.AppCompat.Button.Borderless">
+        <item name="fontFamily">@font/proxima_nova_bold</item>
+        <item name="android:textStyle">bold</item>
+        <item name="android:textSize">16sp</item>
+        <item name="theme">@style/Widget.AppCompat.Button.Borderless</item>
+        <item name="android:layout_gravity">center</item>
+        <item name="android:gravity">center</item>
+        <item name="textAllCaps">false</item>
+        <item name="android:maxLines">1</item>
+        <item name="android:ellipsize">end</item>
+        <item name="android:background">@drawable/btn_socure_orange_corner</item>
+        <item name="android:textColor">@color/socure_title_text</item>
+    </style>
+
+    <style name="socure_consent_privacy_fullscreen">
```

Comment:
We can't expose this consent full screen style in same app. Consent is always full screen, customer can able to alter the screen now. @amar-w